frontend/src/app/Settings/Facility/Components/ManageDevices.tsx

```diff
@@ -117,8 +118,7 @@ const ManageDevices: React.FC<Props> = ({
       <div className="usa-card__footer">
         <Button
           onClick={onAddDevice}
-          type="submit"
-          outline
+          variant="outline"
```

Comment:
The default for a Trussworks button is `button` and not `submit`, which is convenient because you don't need to call `event.preventDefault` in the handler. Instead of having a bunch of individual boolean properties they use a `variant` property to describe the style.